### PR TITLE
Fix capitalization and mask for REX for restorecon example template

### PR DESCRIPTION
**Pull request by maximiliankolb**
I believe that the host sub-status page is no longer accurate but I'd like to not add this to the scope for now. See https://github.com/theforeman/foreman-documentation/issues/2459

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13)

### CODE REVIEW

File: guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc [hunk 1]
```diff
@@ -5,10 +5,34 @@ This example shows how to run a job based on the template created in {ManagingHo
...
-. In the {ProjectWebUI}, navigate to *Hosts* > *All hosts* and select target hosts.
-Select *Schedule Remote Job* from the *Select Action* list.
-. In the *Job invocation* page, select the `Commands` job category and the `Run Command - restorecon` job template.
-. Type `/home` in the *directory* field.
-. Set *Schedule* to `Execute now`.
-. Click *Submit*.
-You are taken to the *Job invocation* page where you can monitor the status of job execution.
+. In the {ProjectWebUI}, navigate to *Monitor* > *Jobs* and click *Run job*.
+. Select *Commands* as *Job category* and *Run Command {endash} restorecon* as *Job template* and click *Next*.
+. Select hosts on which you want to run the job.
```
Comment (mjivraja) [inline guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc:10]:
```suggestion
. Select the hosts on which you want to run the job.
```

File: guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc [hunk 2]
```diff
@@ -5,10 +5,34 @@ This example shows how to run a job based on the template created in {ManagingHo
...
+. In the *directory* field, provide a directory, for example `/home`, and click *Next*.
+. Optional: To configure advanced settings for the job, fill in the *Advanced fields*.
+To learn more about advanced settings, see xref:advanced-settings-in-the-job-wizard_{context}[].
+When you are done entering the advanced settings or you do not need to enter any advanced settings, click *Next*.
+. Schedule time for the job.
+* To execute the job immediately, keep the pre-selected *Immediate execution*.
+* To execute the job in future time, select *Future execution*.
+* To execute the job on regular basis, select *Recurring execution*.
+. Optional: If you selected future or recurring execution, select the *Query type*, otherwise click *Next*.
+* *Static query* means that job executes on the exact list of hosts that you provided.
```
Comment (mjivraja) [inline guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc:21]:
```suggestion
* *Static query* means that the job executes on the exact list of hosts that you provided.
```

File: guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc [hunk 3]
```diff
@@ -5,10 +5,34 @@ This example shows how to run a job based on the template created in {ManagingHo
...
+* *Dynamic query* means that the list of hosts is evaluated just before the job is executed.
+If you entered the list of hosts based on some filter, the results can be different from when you first used that filter.
++
+Click *Next* after you have selected the query type.
+. Optional: If you selected future or recurring execution, provide additional details:
+* For *Future execution*, enter the *Starts at* date and time.
+You also have the option to select the *Starts before* date and time.
+If the job cannot start before that time, it will be canceled.
+* For *Recurring execution*, select the start date and time, frequency, and the condition for ending the recurring job.
```
Comment (mjivraja) [inline guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc:30]:
```suggestion
* For *Recurring execution*, select the start date and time, frequency, and condition for ending the recurring job.
```

File: guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc [hunk 4]
```diff
@@ -5,10 +5,34 @@ This example shows how to run a job based on the template created in {ManagingHo
...
-. Click *Submit*.
-You are taken to the *Job invocation* page where you can monitor the status of job execution.
+. In the {ProjectWebUI}, navigate to *Monitor* > *Jobs* and click *Run job*.
+. Select *Commands* as *Job category* and *Run Command {endash} restorecon* as *Job template* and click *Next*.
+. Select hosts on which you want to run the job.
+If you do not select any hosts, the job will run on all hosts you can see in the current context.
+. In the *directory* field, provide a directory, for example `/home`, and click *Next*.
+. Optional: To configure advanced settings for the job, fill in the *Advanced fields*.
+To learn more about advanced settings, see xref:advanced-settings-in-the-job-wizard_{context}[].
+When you are done entering the advanced settings or you do not need to enter any advanced settings, click *Next*.
```
Comment (mjivraja) [inline guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc:15]:
```suggestion
When you are done entering the advanced settings or if it is not required, click *Next*.
```
Can this be better?

File: guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc]: Applied.

File: guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc (hunk 4 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc]: Applied.

File: guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc (hunk 3 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_executing-a-restorecon-template-on-multiple-hosts.adoc]: Applied.

**Review by maximiliankolb — COMMENTED** [review]
Applied all feedback and rebase to master. Please re-review @mjivraja

**Comment by maximiliankolb** [comment]
Rebased to master to resolve the merge conflict.

**Review by mjivraja — APPROVED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Thanks Malhar.

Merged to master and cherry-picked: :heavy_check_mark: 
   1e55b8eea..6faba2b0f  3.8 -> 3.8
   8c50223a5..a1830cc66  3.7 -> 3.7
   a736d610e..dd6945b86  3.6 -> 3.6
   a42dcdf5b..6d49ee803  3.5 -> 3.5

**Pull request merged**